packages/platform-sdk-ada/src/services/transaction.ts

```diff
@@ -204,13 +246,24 @@ export class TransactionService implements Contracts.TransactionService {
 				  }
 				) {
 				  address
+				  index
 				  transaction {
-					hash
+						hash
 				  }
 				  value
 				}
 			  }`,
 			)
 		).utxos;
 	}
+
+	private async deriveAddressesAndSigningKeys(publicKey: Bip32PublicKey, networkId, accountKey: Bip32PrivateKey) {
```

Comment:
We'll need to figure out how many we need to calculate. In general they suggest using a page size of 20, and then stop when all 20 are unused. But for that we need to issue graphql requests. Which I think is the right thing to do.

Reply:
Don't think we have any other option so go for it.